content/en/database_monitoring/guide/tag_database_statements.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Tagging SQL Statements
+kind: guide
+---
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Database Monitoring is not supported for this site.</div>
+{{< /site-region >}}
+
+This guide assumes that you have configured [Database Monitoring][1].
```

Comment:
Can we put this in an alert-info div? 🙇

Reply:
✅

Reply:
I had to revert this change because the link wasn't rendering properly when using the div 😬 .